app/views/spaces/_website_settings.html.erb

```diff
@@ -4,6 +4,13 @@
     <p class="text-sm italic"><%= t('website_settings.help_text') %></p>
   </header>
 
+  <h4 class="text-base"> Domain </h4>
+  <% if space.branded_domain.present? %>
+    <p class="text-sm"><%= space.branded_domain %></p>
```

Comment:
You could put the `if` inside the `p` to reduce divergence between the branches. Also, `space.branded_domain.presence || "No Domain"` will evaluate to "No Domain" when `space.branded_domain` is `nil` or `#empty?` and the value held in `space.branded_domain` otherwise.

Which means you could do this!

```erb
<p class="text-sm"><%= space.branded_domain.presence || "No Domain"%></p>

```

Reply:
fixed https://github.com/zinc-collective/convene/pull/1600/commits/c5b6ce041bd90d69444aa073276de51e14ab7b44